plugins/kubernetes/app/models/kubernetes/api/pod.rb

```diff
@@ -0,0 +1,26 @@
+module Kubernetes
+  module Api
+    class Pod
+      def initialize(api_pod)
+        @pod = api_pod
+      end
+
+      def ready?
+        @pod.status.phase == 'Running' && condition_ready?
+      end
+
+      def name
+        @pod.metadata.name
+      end
+
+      private
+
+      def condition_ready?
+        @pod.status.conditions.present? &&
+            @pod.status.conditions
+                .select { |c| c['type'] == 'Ready' }
```

Comment:
We should have only one Ready condition: https://github.com/kubernetes/kubernetes/blob/master/docs/user-guide/pod-states.md
I'll update the code.